### PR TITLE
MOB-835: Fix refresh token issue

### PR DESCRIPTION
**Pull request by matir91**
Issue [MOB-835](https://idmeinc.atlassian.net/browse/MOB-835)

### CODE REVIEW

File: me.id.webverify/webverifylib/src/main/java/me/id/webverifylib/AuthToken.java [hunk 1]
```diff
@@ -58,10 +59,14 @@ private boolean isValidToken(String accessToken, Calendar accessTokenExpiration)
...
   public boolean isValidAccessToken() {
-    return isValidToken(accessToken, accessTokenExpiration);
+    return !wasForcedlyInvalidated && isValidToken(accessToken, accessTokenExpiration);
   }
 
   public boolean isValidRefreshToken() {
     return isValidToken(refreshToken, refreshTokenExpiration);
   }
+
+  public void invalidateAccessToken() {
```
Comment (m-revetria) [inline me.id.webverify/webverifylib/src/main/java/me/id/webverifylib/AuthToken.java]: maybe this and the other functions in this class can be package-protected

File: me.id.webverify/webverifylib/src/main/java/me/id/webverifylib/RefreshAccessTokenHandler.java [hunk 2]
```diff
@@ -34,16 +34,18 @@ void refreshAccessToken(IDmeScope scope, AuthToken authToken, IDmeGetAccessToken
         AuthToken token = accessTokenManager.getToken(scope);
         if (token == null) {
           listener.onError(new UnauthenticatedException());
-        } else {
+          return;
+        } else if (token.isValidAccessToken()) {
           listener.onSuccess(token.getAccessToken());
+          return;
```
Comment (bryant1410) [inline me.id.webverify/webverifylib/src/main/java/me/id/webverifylib/RefreshAccessTokenHandler.java]: Why not doing `return` outside of the if (after it)?

Comment (matir91) [inline me.id.webverify/webverifylib/src/main/java/me/id/webverifylib/RefreshAccessTokenHandler.java]: it's an `elseif, it's not the same, right?

Comment (bryant1410) [inline me.id.webverify/webverifylib/src/main/java/me/id/webverifylib/RefreshAccessTokenHandler.java]: oh right, :+1:

**Pull request merged**